frontend/packages/topology/integration-tests/features/topology/topology-application-grouping.feature

```diff
@@ -3,66 +3,41 @@ Feature: Application groupings in topology
-            Given user has created workload "nodejs-ex-git" in application grouping "nodejs-ex-git-app"
-              And user is at the Topolgy page
-             When user clicks on an applicaton grouping "nodejs-ex-git-app"
-             Then user can see application sidebar
+            Given user has created workload "nodejs-ex-git" with resource type "Deployment" and application groupings "nodejs-ex-git-app"
+             When user clicks on application groupings "nodejs-ex-git-app"
+             Then user can see sidebar opens with Resources tab selected by default for application groupings
               And user is able to see workload "nodejs-ex-git" under resources tab in the sidebar
-              And user can see Add to Application and Delete Application in the Action menu
+              And user can see Actions dropdown menu
 
 
-        @smoke @to-do
+        @smoke
         Scenario: Verify Application grouping context menu: T-05-TC02
-            Given user has created workload "nodejs-ex-git" in application grouping "nodejs-ex-git-app"
-              And user is at the Topolgy page
-             When user right clicks on Application "nodejs-ex-git-app" to open context menu
+            Given user is at Topology page
+             When user right clicks on Application "nodejs-ex-git-app" to open Context Menu
              Then user can view Add to Application and Delete Application options
 
 
-        @regression @to-do
+        @regression @todo
         Scenario: Add to Application in Application grouping from Action menu: T-05-TC03
-            Given user has created workload "nodejs-ex-git" in application grouping "nodejs-ex-git-app"
-              And user is at the Topolgy page
-             When user clicks on an applicaton grouping "nodejs-ex-git-app"
-              And user clicks on Action menu and goes to Add to Application
-              And user clicks on From Git
-              And user fills the form and clicks Create
-              And user clicks on Add to Project on Action menu and clicks on Container Image
-              And user fills the form and clicks Create
-              And user clicks on Add to Project on Action menu and clicks on From Dockerfile
-              And user fills the form and clicks Create
-              And user clicks on Add to Project on Action menu and clicks on From Devfile
-              And user fills the form and clicks Create
-              And user clicks on Add to Project on Action menu and clicks on From Event Source
-              And user selects Api Server Source and clicks on Create Event Source
+            Given user is at Topology page
+             When user clicks on application groupings "nodejs-ex-git-app"
+              And user clicks on Action menu
+              And user clicks "Add to Application" from action menu
+              And user clicks on "From Git"
               And user fills the form and clicks Create
```

Comment:
Can you please specify the details including workload name

Reply:
Updated as `user fills the form with workload name "added-application-1" and clicks Create`